2022-07-27 01:11:46,310 : database.py : INFO : The connection to the cassandra driver has been established !!
2022-07-27 01:12:01,390 : database.py : INFO : Establishing keyspace...
2022-07-27 01:12:01,768 : database.py : INFO : The keyspace reviews selected to be used.
2022-07-27 01:12:48,393 : database.py : INFO : Table new_table already exists. Setting the current table to new_table...
2022-07-27 01:13:22,012 : database.py : INFO : Table new_table1 deleted/does not exists from the keyspace reviews
2022-07-27 01:13:40,239 : database.py : ERROR : Error in deleting the table new_table:
 Exact Error: 
errors={'c8a85932-3577-4d27-96ba-1b148c0ae79d-europe-west1.db.astra.datastax.com:29042:a837fe38-9fd5-49a6-958e-7f7ca71ae890': 'Client request timeout. See Session.execute[_async](timeout)'}, last_host=c8a85932-3577-4d27-96ba-1b148c0ae79d-europe-west1.db.astra.datastax.com:29042:a837fe38-9fd5-49a6-958e-7f7ca71ae890
Traceback (most recent call last):
  File "/Users/adityaanuragdash/PycharmProjects/Flipkart_Review_Scrapping/database.py", line 117, in drop_table
    self.session.execute(query)
  File "cassandra/cluster.py", line 2618, in cassandra.cluster.Session.execute
  File "cassandra/cluster.py", line 4894, in cassandra.cluster.ResponseFuture.result
cassandra.OperationTimedOut: errors={'c8a85932-3577-4d27-96ba-1b148c0ae79d-europe-west1.db.astra.datastax.com:29042:a837fe38-9fd5-49a6-958e-7f7ca71ae890': 'Client request timeout. See Session.execute[_async](timeout)'}, last_host=c8a85932-3577-4d27-96ba-1b148c0ae79d-europe-west1.db.astra.datastax.com:29042:a837fe38-9fd5-49a6-958e-7f7ca71ae890
